.istanbul.yml

```diff
@@ -11,7 +9,7 @@ reporting:
     - text
     - lcov
   watermarks:
-    statements: [70, 90]
-    lines: [70, 90]
-    functions: [70, 90]
-    branches: [70, 90]
\ No newline at end of file
+    statements: [50, 80]
```

Comment:
Don't want to bump these watermarks up? I think we should aim for > 90% or >95% coverage on core modules.

Reply:
We can do that in bulk for all repos once they are done.